core/src/main/java/com/scalar/db/api/DistributedTransactionProvider.java

```diff
@@ -3,17 +3,17 @@
 import com.scalar.db.config.DatabaseConfig;
 
 /**
- * A factory class that creates {@link DistributedTransactionManager}, {@link
- * DistributedTransactionAdmin}, and {@link TwoPhaseCommitTransactionManager} instances. Each
- * transaction manager should implement this class to instantiate its implementations of {@link
- * DistributedTransactionManager}, {@link DistributedTransactionAdmin}, and {@link
- * TwoPhaseCommitTransactionManager}. The implementations are assumed to be loaded by {@link
- * java.util.ServiceLoader}.
+ * A class that creates {@link DistributedTransactionManager}, {@link DistributedTransactionAdmin},
+ * and {@link TwoPhaseCommitTransactionManager} instances. Each transaction manager should implement
+ * this class to instantiate its implementations of {@link DistributedTransactionManager}, {@link
+ * DistributedTransactionAdmin}, and {@link TwoPhaseCommitTransactionManager}. The implementations
+ * are assumed to be loaded by {@link java.util.ServiceLoader}.
  */
-public interface DistributedTransactionFactory {
+public interface DistributedTransactionProvider {
```

Comment:
Also renamed `DistributedTransactionFactory` to `DistributedTransactionProvider`.